### PR TITLE
Pb streamlining

### PR DESCRIPTION
**Pull request by bieniusa**
Adding PB Management API and fixes to client as indicated in the corresponding pb codec PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +37,10 @@ AntidoteDB has a simple transactional interface:
 - `commit_transaction/2`
 - `abort_transaction/2`
 
-Here is a detailed interface explanation:
+In the following, you find a short description of the API.
+A more elaborate version with examples can be found in the [AntidoteDB Documentation](https://antidotedb.gitbook.io/documentation/api/protocol-buffer-api).
 
-#### start_transaction(Pid, Timestamp) -> {ok, TxId} | {error, Reason}.
+#### Starting a transaction
```
Comment (peterzeller) [inline README.md:43]: If you remove the signature here, the sentence below starting with "This function" does not fit. Maybe add the signature as a code snippet?

(same below)

File: README.md [hunk 2]
```diff
@@ -7,14 +7,26 @@ This section assumes that you have at least one AntidoteDB node running. Check t
...
 Once you have acquired a `Pid` connection to an AntidoteDB node, you are ready to run transactions.
 
+## Management Interface
+
+For coordinating Antidote instances, you can use the following operations via the Erlang client:
+
+```erl
+Pid = antidotec_pb_socket:start_link("127.0.0.1", 8087).
+%%
+ok = antidotec_pb_management:create_dc(Pid, ["node1", "node2"]).
```
Comment (peterzeller) [inline README.md]: Here you are using Strings, but the type signature below says it should be `[node()]`. I think the type signature is too strict.

File: README.md (hunk 1 above)
Comment (bieniusa) [reply README.md:43]: Done

File: README.md (hunk 2 above)
Comment (bieniusa) [reply README.md]: No, the parameters were wrong.

**Pull request merged**